### DISCUSSION TITLE
Adjust the download link for Microsoft .NET 5.0 Desktop runtime (x86)

### DISCUSSION THREAD

**Discussion by timjobutler** [AntonyCorbett/OnlyM#403] [Ideas]
Hi @AntonyCorbett,

At the locations noted below, users are advised to download the Microsoft .NET 5.0 Desktop runtime (x86) with a note included: "**_Note that it's the desktop x86 version that you need!_**"

- https://github.com/AntonyCorbett/OnlyM#download
- https://github.com/AntonyCorbett/OnlyM/releases/tag/2.0.0.8

As a suggestion, is it possible to remove the note and adjust the current link at the above 2 locations with the direct link to the correct installer which is included below?
https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.8-windows-x86-installer

Hope this small change can reduce any chances of confusion with downloading correct version of Microsoft .NET 5.0 runtime. Thanks for considering.

Your brother,
Tim

**Comment by richfrantz** [comment]
+1
took me too long to figure out why the x64 was not working even after multiple attempts and reboots.
________________________________
From: Tim Butler ***@***.***>
Sent: Saturday, August 7, 2021 4:29 PM
To: AntonyCorbett/OnlyM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AntonyCorbett/OnlyM] Adjust the download link for Microsoft .NET 5.0 Desktop runtime (x86) (#403)


Hi @AntonyCorbett<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAntonyCorbett&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523563535%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=7R6O6K1Gj9mTvfm34nHGEp5Tf73UbrPzgsEB1Vt0v2s%3D&reserved=0>,

At the locations noted below, users are advised to download the Microsoft .NET 5.0 Desktop runtime (x86) with a note included: "Note that it's the desktop x86 version that you need!"

  *   https://github.com/AntonyCorbett/OnlyM#download<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAntonyCorbett%2FOnlyM%23download&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523563535%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=MNCHzHVzScZIuhTwUvU%2F1v17M5dqbdsZ9x7CtpcpMd8%3D&reserved=0>
  *   https://github.com/AntonyCorbett/OnlyM/releases/tag/2.0.0.8<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAntonyCorbett%2FOnlyM%2Freleases%2Ftag%2F2.0.0.8&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523573492%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=CUNRh%2Buem5PhuJ6l4epjhksbz3PfP1ZU46wYe16pTXs%3D&reserved=0>

As a suggestion, is it possible to remove the note and adjust the current link at the above 2 locations with the direct link to the correct installer which is included below?
https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.8-windows-x86-installer<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdotnet.microsoft.com%2Fdownload%2Fdotnet%2Fthank-you%2Fruntime-desktop-5.0.8-windows-x86-installer&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523573492%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=c8Q7XHyc3rRBgY2%2FPrQuABbPlNNJuI7xn%2BygYo3ffJk%3D&reserved=0>

Hope this small change can reduce any chances of confusion with downloading correct version of Microsoft .NET 5.0 runtime. Thanks for considering.

Your brother,
Tim

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAntonyCorbett%2FOnlyM%2Fdiscussions%2F403&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523583482%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=VDLhRGsVRrLCqk9CKOYMUqK3LVsv5qyRvIkMXk4b94c%3D&reserved=0>, or unsubscribe<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAINW6PJYPBZAJLOFUFQ7GXDT3W6ULANCNFSM5BX3UK4Q&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523583482%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=EiNK7z0EQeFFD1COVuwXbzeFE7vI2fux1l4cBNmwsr4%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523593397%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=sths%2FtNjxXD4xU9G%2BlsNnv2bHG6uLiiLySh0E11KdMU%3D&reserved=0> or Android<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7C%7C2319b2d299a84a1a40c108d959fb28f8%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637639757523593397%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=gK1%2BWUULvzQZcukB3cOgPzMTS0u%2Bguvosiu87nM8RUc%3D&reserved=0>.

**Comment by AntonyCorbett** [comment]
@timjobutler Thanks for the suggestion. I'd need to keep updating the reference as the .NET 5 version is incremented. There isn't yet a generic direct link to the _latest_ release of each variant. I've tried to emphasise several times which installer is needed. I will try to include logic in the OnlyM installer to run the .NET 5 installer if needed.

**Reply by timjobutler** [reply]
@AntonyCorbett That is a fair point and would create too much extra wok to keep updating the links every time that the .NET 5 installer version is updated.

Adding logic to the OnlyM installer is great solution. Happy to have this item closed.
